package com.thinkinginjava.error_handling_with_exceptions.example.chapter12_9;

/**
 * page 269
 *   -- chapter_12 通过异常处理错误
 *
 *   12.9 异常的限制
 */